sdcm/tester.py

```diff
@@ -2926,10 +2919,22 @@ def disable_table_encryption(self, table, upgradesstables=True):
         self.alter_table_encryption(
             table, scylla_encryption_options="{'key_provider': 'none'}", upgradesstables=upgradesstables)
 
-    def alter_test_tables_encryption(self, scylla_encryption_options=None, upgradesstables=True):
-        for table in self.db_cluster.get_non_system_ks_cf_list(self.db_cluster.nodes[0], filter_out_mv=True):
-            self.alter_table_encryption(
-                table, scylla_encryption_options=scylla_encryption_options, upgradesstables=upgradesstables)
+    def alter_test_tables_encryption(self, upgradesstables=True, stress_command=''):
+        """
+        Configure encryption at-rest for all test tables if needed,
+        sleep a while to wait the workload starts and test tables are created
+        """
+        append_scylla_yaml = yaml.safe_load(self.params.get("append_scylla_yaml") or '') or {}
+
+        if (scylla_encryption_options := self.params.get('scylla_encryption_options')
```

Comment:
This walurs operator, probably doesn't work as I thought it would
And the variable is set to True, and not to encryption option as I intended

@roydahan @k0machi